lib/cucumber/formatter/ansicolor.rb

```diff
@@ -83,17 +83,17 @@ module ANSIColor
         next if method_name =~ /.*_param/
         code = <<-COLOR
           def #{method_name}(string=nil, &proc)
```

Comment:
This is a really crappy way of doing this metaprogramming, we should be using define method, but maybe one for another time.

it also fixes the security eval offense (But can defer), maybe add this to the rubocop todo file

Reply:
Yeah, I just fix offenses. But that whole file ... I think it need a proper refactoring.
But for another PR?

Reply:
Happy for that.